.changeset/clean-lemons-jump.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-components': minor
```

Comment:
```suggestion
'@backstage/core-components': patch
```

Gonna switch this back to a patch because I don't think this quite warrants a breaking bump. Minor bumps are for breaking changes since this package is on version 0.x